### ISSUE TITLE
[Bug][macOS]: Dragging from Latest Episodes to the end of Up Next doesn't place the item at the end of the list

### ISSUE THREAD

**Issue by Shamino0** [gdavis/Mimir-Feedback#254]
### Are you sure this is a new issue?

- [X] I have confirmed this is a new issue

### macOS Version

Sonoma 14.3

### Your Mac

Mac mini (2018)

### App Version

1.8.2 (159)

### What happened?

Dragging an episode from the Latest Episodes list (center column) to the end of the Up Next list (to append it to the list) ends up inserting the episode as second on the list instead of putting it at the end.

Once this is done, I can manually re-order the Up Next list.

### Bug prevalence

Every day

### How does it happen?

Start from a state where there are several episodes in the Latest Episodes list (center column) and an empty Up Next list (right column)

1. Drag an episode (let's call it "A") from Latest to Up Next.  It gets added to the list.  Up Next now has just "A".
2. Drag another episode ("B") from Latest to the bottom of the Up Next list.  It gets added to the bottom.  Up Next now shows "A, B".
3. Drag a third episode ("C") to the bottom of the Up Next list.  It doesn't get added to the bottom, but ends up in the second position.  Up Next now shows "A, C, B", when it should be "A, B, C"
4. Drag a fourth episode ("D") to the bottom of the Up Next list.  It also adds to the second position.  Up Next now shows "A, D, C, B" when it should be "A, B, C, D"

**Comment by gdavis** [comment]
Hello @Shamino0 and thanks for taking the time to make this report. I've been working on this today and found a fix for it that will be in the next release. 🚀

**Comment by Shamino0** [comment]
Confirmed fixed in version 1.8.3 (162).  Thanks much for your work on this.
